src/flownet/static/SUMMARY.inc

```diff
@@ -1,105 +1,155 @@
+-- ######################
+-- ## Field quantities ##
+-- ######################
+
+-- Production rates
 FOPR
-FOPT
+FGPR
+FWPR
+FLPR
+FVPR
 FOPRH
-FOPTH
-FWPT
-FWPTH
+FGPRH
+FWPRH
+FLPRH
+FVPRH
```

Comment:
Quickly scanning through the tables in the OPM manual, and it seems like FVPRH is not supported. Other than that it looks all good.